src/modules/lists/lists.controller.ts

```diff
@@ -106,4 +141,39 @@ export class ListsController {
       success: true,
     };
   }
+
+  @ApiOperation({ title: `Deletes the given mentor's list for the given user` })
+  @Delete(':listId')
+  async deleteList(
```

Comment:
@crysfel Should I add `@UsePipes(new ValidationPipe({ transform: true, whitelist: true }))` here? When should we add that and when not to? Since the @Get method is not using it.

Reply:
The validation pipe validates the body's payload, in this particular endpoint we are not sending a body payload, therefore there's no need to validate anything (same as the GET method).

Basically, you will add validations whenever you need to validate the body's payload, such as in POST or PUT where we are adding/updating the record

Reply:
I see, thanks for the explanation